src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -82,12 +82,6 @@ public bool GenerateFullPaths
             get { return _store.GetOrDefault(nameof(GenerateFullPaths), false); }
         }
 
-        public string LangVersion
```

Comment:
@agocke Yes, I suspect it is. But if I infer the verb in Jared's response ;-), I think the DLL isn't tracked for binary compat (only tasks and targets).

That said, I can modify the PR to avoid this (create an protected "GetLanguageVersion()" on the base task, and leave the language-specific LanguageVersion properties where they were), if this is controversial.